static/app/components/codeSnippet.tsx

```diff
@@ -36,6 +36,7 @@ interface CodeSnippetProps {
    */
   isRounded?: boolean;
   language?: string;
+  linesToHighlight?: number[];
```

Comment:
Since this is a highly used component, could you do a couple extra things:

1. Add a comment here explaining what the prop does
2. Add section to the `codeSnippet.stories.tsx` file that demos the functionality. You can view the story book at `/stories`

Reply:
Done in next commit